contracts/BasePaymaster.sol

```diff
@@ -26,10 +26,13 @@ abstract contract BasePaymaster is IPaymaster, Ownable {
         return address(relayHub);
     }
 
-    // Gas stipends for acceptRelayedCall, preRelayedCall and postRelayedCall
-    uint256 constant private ACCEPT_RELAYED_CALL_GAS_LIMIT = 50000;
-    uint256 constant private PRE_RELAYED_CALL_GAS_LIMIT = 100000;
-    uint256 constant private POST_RELAYED_CALL_GAS_LIMIT = 110000;
+    //Paymaster is commited to pay for any reverted transaction above the commitment. it covers both preRelayedCall aod
+    // forwrader's check for nonce/signature.
+    // we assume 50k is more than enough for forwarder (10k bytes request takes ~30kgas)
+    uint256 constant public COMMITMENT_GAS_LIMIT = 150000;
```

Comment:
Please document the importance of the diff between commitment and pre gas. Also add tests for that.